### PR TITLE
feat(GitHub): Add Get Closed PR by repo

### PR DESCRIPTION
**Pull request by FlorentLvr**

### CODE REVIEW

File: GitHub/GitHub_Get_Closed_PR_by_repo.ipynb [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+   "id": "21788077-c209-4ab5-b047-7ce1b2f13e0f",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "from github import Github\n",
```
Comment (srini047) [inline GitHub/GitHub_Get_Closed_PR_by_repo.ipynb:105]: Screenshot 2023-02-24 at 6 42 20 PM

Getting Import error. Also this doesn't work in Python 3.7+ as per this [comment](https://github.com/PyGithub/PyGithub/issues/856#issuecomment-453809986).

If you are fine then let's just get the PR merged.

**Review by srini047 — COMMENTED** [review]
@FlorentLvr Any ideas on how to move forward with it.

File: GitHub/GitHub_Get_Closed_PR_by_repo.ipynb (hunk 1 above)
Comment (FlorentLvr) [reply GitHub/GitHub_Get_Closed_PR_by_repo.ipynb:105]: @srini047, I just tested it again and it works on my Naas. Let me check on your naas account why it is not working.

Comment (srini047) [reply GitHub/GitHub_Get_Closed_PR_by_repo.ipynb:105]: @FlorentLvr Any updates on this?

**Pull request merged**